netbox/templates/inc/panels/custom_fields.html

```diff
@@ -12,11 +12,16 @@ <h6>{{ group_name }}</h6>
           <table class="table table-hover attr-table">
             {% for field, value in fields.items %}
               <tr>
-                <th scope="row">
-                  <span title="{{ field.description|escape }}">{{ field }}</span>
-                </th>
```

Comment:
I think we still want to show the field name, but replace the `title` attribute with an icon within the same cell.

Reply:
Could you please clarify a bit more? The field name is still showing up here

![image](https://user-images.githubusercontent.com/5083532/236628317-dc5c7387-e3e9-4a24-b532-097d3915c210.png)
![image](https://user-images.githubusercontent.com/5083532/236628321-0037751b-0fd9-4570-9584-2482b40404a4.png)